lib/LaTeXML/Engine/LaTeX.pool.ltxml

```diff
@@ -2045,15 +2045,9 @@ DefConstructorI('\lx@begin@display@math', undef,
     . "</ltx:Math>"
     . "</ltx:equation>",
   alias        => '$$',
-  beforeDigest => sub {
-    $_[0]->beginMode('display_math');
-    if (my @everymath_toks = $STATE->lookupDefinition(T_CS('\everymath'))->valueOf->unlist()) {
-      $_[0]->getGullet->unread(@everymath_toks); }
-    if (my @everydisplay_toks = $STATE->lookupDefinition(T_CS('\everydisplay'))->valueOf->unlist()) {
-      $_[0]->getGullet->unread(@everydisplay_toks); }
-    return; },
-  properties  => sub { RefStepID('equation') },
-  captureBody => 1);
+  beforeDigest => sub { $_[0]->beginMode('display_math'); },
```

Comment:
Very nice simplification. The cherry on the cake would be to switch the few cases of 
```
beforeDigest => sub { $_[0]->beginMode('display_math'); }
``` 

to the abbreviated form of 

```
mode => 'display_math'
```

Reply:
Ha! I fell into that trap! (& wasted time). It's subtly different. That short form simply switches the mode (typically within an environment which establishes a grouping), whereas the commands in question like `\lx@begin@displaymath` come in pairs which will do `beginMode` and `endMode`.

Reply:
Ah! `mode` also ends the mode at the end, indeed that slipped my mind. Apologies.